### ISSUE TITLE
Bountysource

### ISSUE THREAD

**Issue by intermaxim** [acceptbitcoincash/acceptbitcoincash#255]
*Please fill in the template below, and don't worry about exact formatting, just do your best!

Requesting to add {{ Bountysource }} to the {{ developer }} category. Details follow:

```yml

- name: Bountysource
  url: https://www.bountysource.com/
  twitter: bountysource
  facebook: Bountysource
  bcc: no
  btc: yes
  othercrypto: no
```

Add more information here if necessary

**Comment by intermaxim** [comment]
I forgot to include their e-mail: support@bountysource.com

**Comment by t0p0lin0** [comment]
Added and merged in with pull request #273 -- thank you for contributing!
